core/platform/models_test.py

```diff
@@ -273,3 +273,8 @@ def test_import_search_services(self):
         self.assertEqual(
             self.registry_instance.import_search_services(),
             gae_search_services)
+
+    def test_import_models_not_implemented_has_not_implemented_error(self):
+        """Tests NotImplementedError of Platform."""
+        with self.assertRaises(NotImplementedError):
+            _platform_instance = models.Platform().import_models()
```

Comment:
You can drop the `_platform_instance =` part, since it's not being used.

Reply:
Done